1-js/10-error-handling/1-try-catch/article.md

```diff
@@ -623,53 +623,53 @@ For instance:
 */!*
 
   function readData() {
-    badFunc(); // Whoops, something went wrong!
+    badFunc(); // !اوه، یک جای کار می‌لنگد
```

Comment:
More attention here.

Reply:
```suggestion
    badFunc(); // !اوه، یه جای کار می‌لنگه
```

Reply:
> 

I think the formal form is better.  
Thanks for your suggestion, though.